### ISSUE TITLE
64x128 panel overlapping LEDs

### ISSUE THREAD

**Issue by I3lackshirts** [hzeller/rpi-rgb-led-matrix#1282]
![F273E6E3-70A9-43CC-8333-174685EAA1A0](https://user-images.githubusercontent.com/81433751/112663026-f628d900-8e1d-11eb-8ad4-f300531ba198.jpeg)

see the attached picture. Basically what is happening is the first 16 rows are displaying both the first 16 rows and rows 17-32 on top of each other. Same thing is happening for the bottom half. I tried inputting a different led-panel-type and just was getting errors. Anyone run into this before and know what to try?

**Comment by davemaster** [comment]
Greetings,

This a chain or parallel or panel (a picture of its back will be usefull) issue.

What is the command are You using with this output? Can You share it?

Best regards

**Comment by hzeller** [comment]
So this looks like a 64 pixel high panel. Did you invoke it with `--led-rows=64`, and, if you happen to use one of the Adafruit HATs, did the [necessary hardware modification](https://github.com/hzeller/rpi-rgb-led-matrix#64x64-with-e-line-on-adafruit-hatbonnet) ?

**Comment by I3lackshirts** [comment]
So I am using the Electro dragon RGB panel v2 https://www.electrodragon.com/product/rgb-matrix-panel-drive-board-for-raspberry-pi-v2/ so I am thinking it will need a similar hardware modification? There is not much documentation about this board so not sure where to look to see where I can make a similar modification

**Comment by davemaster** [comment]
> 
> 
> So I am using the Electro dragon RGB panel v2 https://www.electrodragon.com/product/rgb-matrix-panel-drive-board-for-raspberry-pi-v2/ so I am thinking it will need a similar hardware modification? There is not much documentation about this board so not sure where to look to see where I can make a similar modification
> 
> And it is not a chain. Its one panel 64x128. I did use the commands to specify rows=64 cols=128 and no hardware pulse.

NO, use "regular" for work with that adapter. But, also, share your command, the one produce the output in your attached picture

**Comment by hzeller** [comment]
The electrodragon is similar to the adapter in this project. If you have a 64 pixel high panel, then you need to set a jumper; they have a similar arrangement like in [the rpi-rgb-matrix adapter](https://github.com/hzeller/rpi-rgb-led-matrix/tree/master/adapter/active-3#choose-e-line-for-64x64-panels-with-132-multiplexing)

**Comment by marcmerlin** [comment]
With apologies to everyone, this project is doing a one time cleanup of 
all the bugs that have not been addressed and closed in the last 10 years.

If you filed an issue to ask for help, please do not re-open this one or a new
bug, but go here instead: https://rpi-rgb-led-matrix.discourse.group/

Please be mindful that this project does not have any full time maintainers
or anyone paid. You should not expect personal help for your own projects or
answers to programming questions but you can ask for help with panels that
aren't working, setup questions, or anything that may be a bug here:
https://rpi-rgb-led-matrix.discourse.group/
If you need programming help and are willing to pay someone for it, please
state so in your message, and someone may be interested.

If you did report an actual bug that is still not fixed today, please accept
our apologies for this being mass closed, you can absolutely send a followup
and re-open the issue, or open a new up to date one as applicable.
Note that we very much welcome help, so if you can help fix your bug, you will
be a superstar :)

If your bug was linked to a pull request that was not resolved, please update
the pull request and the issue, and we will do our best to review them. We do
thank you for your undertanding that we have very limited free time, so any
extra work you do along with your bug report or PR request will be very much
appreciated and improve the odds that they will be looked at.

If you want to help contribute as a python or C# or any other language bindings,
you are also a superstar, please state so in the issue and we'll do our best to
give you the access and reviews you need.

For anything related to PWM panel support, please make sure you read README and
then refer to https://github.com/hzeller/rpi-rgb-led-matrix/issues/466 . 
Please do not comment if you're haven't read everything and/or if you're just
asking when your panel is going to be supported. Those bugs are if you're 
able to contribute to the effort.

For rPI5: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1603

For anything python, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
For anything C#, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
